draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -622,26 +596,26 @@
    ----------------+
 					</artwork>
 				</figure>
-				<t>The algorithm starts with cwin = LPP + 1. In the congestion avoidance phase, cwin increases as LSPs are acked: for every acked LSP, cwin += 1 / cwin. Thus, the sending rate is inversely proportional to the RTT. Since the RTT is low in many IS-IS deployments, the sending rate can reach fast rates in short periods of time.</t>
+				<t>The algorithm starts with cwin = cwin0 = LPP + 1. In the congestion avoidance phase, cwin increases as LSPs are acked: for every acked LSP, cwin += 1 / cwin. When LSPs are exchanged, cwin LSPs will be acknowledged in 1 RTT, meaning cwin(t) = t/RTT + cwin0. Since the RTT is low in many IS-IS deployments, the sending rate can reach fast rates in short periods of time.</t>
 
 				<t>When updating cwin, it must not become higher than the number of LSPs waiting to be sent, otherwise the sending will not be paced by the receiving of acks. Said differently, tx pressure is needed to maintain and increase cwin.</t>

```

Comment:
Can we also add  `cwin, by definition, should not become bigger than the receive window advertised by the receiver.` ?

Reply:
Yes. Feel a priory obvious but that's part of the spec and better safe than sorry.
Changed to "for every acked LSP, cwin += 1 / cwin without exceeding RWIN.". Please feel free to comment/correct.